sig-and-wg/sig-core/proposals/installer-release.md

```diff
@@ -0,0 +1,31 @@
+# Kyma Installer with embedded Kyma charts and Tiller
+
+Created on 2018-09-19 by Piotr Bochyński (@pbochynski).
+
+## Status
+
+Proposed on 2018-09-19
+
+## Context
+
+Current installation process involves downloading Kyma charts from the cloud storage. Such solution allows to release Kyma installer and kyma charts independently. In the consequence, there is possibility to install different versions (or flavors) of Kyma components using a single installer binary. It seems to be a convenient feature but in reality, it introduces the following problems:
+- Build and release processes are more complex since they require publishing Kyma charts in the cloud storage.
+- It is possible to run the installer with the Kyma charts version which is not supported and not tested with the installer.
+- The upgrade process is more complex. Usually, the installer upgrade must precede the Kyma release URL update.
+
+## Proposal
+
+Installer will contain Kyma charts and scripts in the Docker image. The build process must be adapted to produce the installer image for every Kyma snapshot or release.
+
+Installer deployment will also contain tiller container that will be used to install helm charts. This change should be applied also in helm-broker. When it is done tiller is no longer required as a prerequisite.
```

Comment:
``The installer deployment will also contain a Tiller container that will be used to install Helm charts. This change should also be applied in the Helm Broker. When it is done, Tiller will no longer be required as a prerequisite.``

Reply:
Removed from proposal.